.github/workflows/ci_workflows.yml

```diff
@@ -58,6 +58,11 @@ jobs:
             python: 3.9
             toxenv: py39-test
 
+          - name: Python 3.10 with minimal dependencies
```

Comment:
Originally when I attempted this, I put this in devdeps job.

I also wonder if "minimal dependencies" is enough to catch everything...

Reply:
Also would need a follow up issue to test with proper Python 3.10 when it is released for real.

Reply:
Yes we'll need to update for rc1 and the final release.
I didn't try with all other dependencies because I'm afraid this will fail somewhere. But it's fine for now, I mean the goal here is to test that we are compatible with 3.10, not to test that other dependencies also are.

Reply:
Long term I think it's worth bundling this with dev dependencies, especially if they provide a nightly build with dev/rc python.